app/app/explore/Explore.tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+
+import { useState } from "react";
+import LiquidityPools from "./LiquidityPools";
+import SaverVaults from "./SaverVaults";
+import { PoolDetails, StatsData } from "@/midgard";
+
+export interface Saver {
+  asset: string;
+  saversCount: number;
+  saversReturn: string;
+  earned: string;
+  filled: number;
+  assetPriceUSD: string;
+  saversDepth: string;
+  assetDepth: string;
+  synthSupply: string;
+}
+
+interface ExploreProps {
+  pools: PoolDetails;
+  stats: StatsData;
+  savers: Saver[];
+}
+
+const Explore = ({ pools, stats, savers }: ExploreProps) => {
+  const [activeTab, setActiveTab] = useState("liquidity");
+  const runePriceUSD = parseFloat(stats.runePriceUSD);
+
+  return (
+    <main className="mx-auto">
+      <nav className="mb-4">
+        <ul className="flex space-x-8 font-semibold font-gt-america-exp text-2xl">
+          <li>
+            <a
+              href="#"
+              onClick={() => setActiveTab("liquidity")}
```

Comment:
imo i think we should have pools and savers as different pages, but we can refactor later